test/snapshot/byref.test.js

```diff
@@ -33,9 +33,6 @@ describe("byref", () => {
   it("function definition", () => {
     expect(parser.parseEval("function &foo( &$bar ) { }")).toMatchSnapshot();
   });
-  it("variable", () => {
```

Comment:
why removing this test ?

Reply:
the jest-eslint-plugin revealed that it's a duplicate.